plugins/source/gcp/client/spec.go

```diff
@@ -19,4 +20,9 @@ func (spec *Spec) setDefaults() {
 	if spec.FolderRecursionDepth == nil {
 		spec.FolderRecursionDepth = &defaultRecursionDepth
 	}
+
+	var defaultDiscoveryConcurrency = 100
+	if spec.DiscoveryConcurrency == nil {
+		spec.DiscoveryConcurrency = &defaultDiscoveryConcurrency
```

Comment:
You could define `defaultDiscoveryConcurrency` inside the block.

Reply:
I was trying to match the style in lines 19-21...